controllers/comments.controllers.js

```diff
@@ -0,0 +1,21 @@
+const { checkArticleIdExists } = require("../models/articles.models");
+const { selectAllCommentsForArticle } = require("../models/comments.models");
+
+exports.getAllCommentsForArticle = (req, res, next) => {
+    const { article_id } = req.params;
+
+    const promises = [selectAllCommentsForArticle(article_id)];
+
+    if (article_id) {
```

Comment:
Do we need to check if there is an article id? What happens if there isn't?

Reply:
In order to implement that suggestion I added a separate test for the following path - "/api/articles//comments", where no article_id is provided. It will be handled as a custom error.